include/oneapi/dpl/pstl/utils_ranges.h

```diff
@@ -272,7 +272,8 @@ struct reverse_view_simple
     }
 };
 
-//It is kind of pseudo-view for take_view support.
+//It is kind of pseudo-view for take_view support. We assume that the underlying range will not shrink
+//after creation of the view to favor performance.
```

Comment:
The comments need to be reverted as well as they are no longer the approach / assumption.